spec/index.html

```diff
@@ -600,7 +600,13 @@ <h3>IRIs</h3>
         <dfn data-lt="URI" data-lt-noDefault><abbr title="Uniform Resource Identifier">URI</abbr>s</dfn>
         [[RFC3986]] that permits a wider range of Unicode characters.
         Every absolute URI and URL is an IRI, but not every IRI is an URI.
-        In RDF, IRIs are used as <em>IRI references</em>, as defined in [[RFC3987]].
+        In RDF, IRIs are used as <dfn data-lt="iri reference">IRI references</dfn>, as defined in [[RFC3987]]
+        <a data-cite="RFC3997#section-1.3">section 1.3</a>.
+        An IRI reference is common usage of an Internationalized Resource Identifier.
+        An IRI reference may be absolute or relative.
+        However, the <a>IRI</a> that results from such a reference
+        only includes absolute IRIs; any <a>relative IRI references</a> are
+        resolved to their absolute form.
```

Comment:
Not true - there's a catch.

https://datatracker.ietf.org/doc/html/rfc3986#appendix-D.1

An IRI reference is either an "IRI" or a " relative reference".
```
   URI-reference = URI / relative-ref
```

By spec, absolute means "no fragment" - i.e. the form used HTTP request lines.

The correct term is simply "IRI"

```
   URI-reference = URI / relative-ref
   absolute-URI  = scheme ":" hier-part [ "?" query ]
   relative-ref  = relative-part [ "?" query ] [ "#" fragment ]
```

Reply:
From [RFC3987 section 1.3](https://datatracker.ietf.org/doc/html/rfc3987#section-1.3):

> IRI reference: Denotes the common usage of an Internationalized
      Resource Identifier.  An IRI reference may be absolute or
      relative.  However, the "IRI" that results from such a reference
      only includes absolute IRIs; any relative IRI references are
      resolved to their absolute form.  Note that in [[RFC2396](https://datatracker.ietf.org/doc/html/rfc2396)] URIs did
      not include fragment identifiers, but in [[RFC3986](https://datatracker.ietf.org/doc/html/rfc3986)] fragment
      identifiers are part of URIs.

Given that the term and usage of "IRI reference" isn't going to just go away, we should say something about it, rather than leaving it to RFC3987. By definition an "IRI" is absolute, and a "relative IRI reference" is relative. When syntaxes allow either form, the term "IRI reference" is useful. Do we need further clarification that an IRI reference ultimately resolves to an (absolute) IRI?

```
IRI            = scheme ":" ihier-part [ "?" iquery ]
                     [ "#" ifragment ]
IRI-reference  = IRI / irelative-ref
irelative-ref  = irelative-part [ "?" iquery ] [ "#" ifragment ]
```


These all allow fragments.

Reply:
RFC 3987 is inconsistent. The grammar of 3987 says: `absolute-IRI   = scheme ":" ihier-part [ "?" iquery ]`. That's the nearest to a definition of absolute IRI. It is only the text in 3987 "to their absolute form" that suggests otherwise. 

RFC 3987 does not replace 3986. 3987 does not claim to do more than widen the characters in certain places.

[RFC 3986 section-4.3](https://datatracker.ietf.org/doc/html/rfc3986#section-4.3) is a section about absolute URIs.

[RFC 3986 D.1](https://datatracker.ietf.org/doc/html/rfc3986#appendix-D.1)

> An ABNF rule for URI has been introduced to correspond to one common
> usage of the term: an absolute URI with optional fragment.



> Do we need further clarification that an IRI reference ultimately resolves to an (absolute) IRI?

An IRI reference resolves to an IRI.
```
IRI-reference  = IRI / irelative-ref
```

Reply:
Clarified in https://github.com/w3c/rdf-concepts/pull/57/commits/d26528cf4070ebb1932c6b3d0e8946d7e1943441.